CheckerUI_xforms/CheckerUI/CheckerUI/CheckerUI/Views/LineView.xaml.cs

```diff
@@ -30,11 +27,29 @@ public LineView(string i_Title)
             baseVM.init(m_OrdersLayout);
             BindingContext = baseVM;
             m_GetOrdersButton.Command = baseVM.FeelOrdersCommand;
+            m_ReturnButton.Command = baseVM.ReturnCommand;
         }
-        private void toMakeView_OnItemSelected(object sender, SelectedItemChangedEventArgs e)
+        protected async override void OnAppearing()
+        {
+            base.OnAppearing();
+            // this is the only place of initiating the connection
+            if (App.HubConn.State == HubConnectionState.Disconnected)
+            {
+                try
+                {
+                    await App.HubConn.StartAsync();     // start async connection to SignalR Hub at server
+                    await App.HubConn.InvokeAsync("InitialToDos");  // invoke initial event - to get all current listings
+                }
+                catch (System.Exception ex)
+                {
+                    Console.WriteLine(ex.Message);
+                }
+            }
```

Comment:
While this activates the connection on appearance of the line view, but the initial event for the hub is already activated in Order Manager, which is supposed to rise before any lineview is accessible, no?